src/Build/Logging/SerialConsoleLogger.cs

```diff
@@ -91,6 +91,7 @@ internal override void ResetConsoleLoggerState()
             errorCount = 0;
             warningCount = 0;
 
+            propertyOutputMap = new Dictionary<(int nodeId, int contextId), StringBuilder>();
```

Comment:
I might go so far as to say that if you're using the serial console logger you should just not get the new behavior. Is that too conservative?

Reply:
I think using the serial console logger you could still be multitargetted or using various properties that could help you debug a build. Do you think it would not be as helpful to people or more that it would break something for people?